<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvertWindow</class>
 <widget class="QWidget" name="ConvertWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Конвертирование</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Конвертировать в:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_convert_jpg">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>jpg-файл</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_convert_xlsx">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>xlsx-файл</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_convert_csv">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>csv-файл</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
